concent_api/core/tests/test_unit_message_handlers.py

```diff
@@ -64,3 +69,24 @@ def test_that_messages_are_stored_with_correct_timestamps(self):
             parse_iso_date_to_timestamp(subtask.report_computed_task.timestamp.isoformat()),
             parse_iso_date_to_timestamp(self.report_computed_task_timestamp)
         )
+
+
+class TestIfObjectIsEnumInstance(TestCase):
+
+    def test_that_validation_passes_if_correct_enum_instance_given(self):  # pylint: disable=no-self-use
+        validate_if_given_object_is_expected_enum_instance(constants.ErrorCode.MESSAGE_INVALID, constants.ErrorCode)
+
+    def test_that_validation_raises_http400_if_string_given(self):
+        with self.assertRaises(GolemMessageValidationError):
+            validate_if_given_object_is_expected_enum_instance('any_string', constants.ErrorCode)
+
+    def test_that_validation_raises_http400_if_other_enum_given(self):
+        with self.assertRaises(GolemMessageValidationError):
+            validate_if_given_object_is_expected_enum_instance(constants.ErrorCode.MESSAGE_INVALID, PendingResponse.ResponseType)
+
+    def test_that_validation_raises_http400_if_none_given_and_none_not_allowed(self):
```

Comment:
Those are not http400 any more, do they? :D Just change to `test_that_validation_raises_exception_...`

Reply:
As always right :D